### DISCUSSION TITLE
NCIRF Beta 3.0

### DISCUSSION THREAD

**Discussion by choonsiklee** [ncidose/ncidosetools#37] [Announcements]
We're excited to release the user beta version of NCIRF3.0 after rigorous alpha testing! In this version, you can use a total of 351 pediatric and adult phantoms with various heights and weights just like the NCICT size-dependent phantom library. Please give it a try and post your feedback.

There is the second tab on the phantom library section as below. You need to select the age group (pediatric or adult) and gender (male or female). Then you can use the small up and down arrows to change the height and weight. You can refer to the phantom library map by clicking "Phantom Height Weight Map" button.

<img width="274" alt="image" src="https://github.com/ncidosimetry/ncidoseforum/assets/22055904/50907818-a3e0-4809-b22d-e4fd06d90de6">

You can use two types of Batch Manager: one for reference-size phantoms and the other for non-reference size phantoms. Depending on the phantom tab you are on (reference or non-reference), different Batch Manager will be launched. In the Batch Manager for non-reference phantoms, the 2nd, 3rd, and 4th input parameters include age group (1=pediatric female, 2=pediatric male, 3=adult female, 4=adult male), height (cm), and weight (kg).

<img width="1000" alt="image" src="https://github.com/ncidosimetry/ncidoseforum/assets/22055904/61f9d601-8298-457a-8cae-ddb0c1aa42a9">

**Comment by angelogiannone** [comment]
Hi,
I've tried NCIRF 3.0beta on both Mac and Windows, but the same error keep appearing:
![Schermata 2024-03-20 alle 09 41 44](https://github.com/ncidosimetry/ncidoseforum/assets/145598541/adf25e8a-b453-4e1d-a027-4c223d124e11)
I know it can be dependent on whether there are spaces in the username, but it's not the case on my Mac, how can I solve this?
Thanks,
AG

**Reply by choonsiklee** [reply]
@angelogiannone Do you still have this problem? We have tested on multiple Mac and Window machines but do not experience this issue... please send us more detailed description of the problem and we will check again. Thanks!

**Reply by angelogiannone** [reply]
Hi Dr. Lee, I tested on different machines too and I think the problem can be related to the MacOS version (10.15)

**Comment by choonsiklee** [comment]
From @angelogiannone 

Good morning dr. Lee, 

We have recently conducted tests on NCIRF Beta 3.0, and we would like to share some feedback with you. Firstly, we must commend the software for its array of phantom options and its automated matching of beam quality and phantom.

However, during our evaluation, we observed a peculiar behavior in the coordinate system concerning the pediatric non-reference phantoms. It appears that certain phantoms have been scaled up to a height of 1.70 m, which, if our understanding of the software is correct, could pose challenges for simulations, particularly when performed in batch mode. This is primarily due to issues related to isocenter placement and field dimensions.

Attached are screenshots (1 and 2) illustrating that a 10x10cm field covers only a portion of the pediatric non-reference phantom's forehead (screenshot 1), whereas in the corresponding reference phantom (screenshot 2), it covers a significantly larger surface area. This discrepancy suggests a potential scaling factor at play. Subsequently, screenshots (3 and 4) demonstrate how a field with height equivalent to the phantom's height encompasses the entire phantom in the reference case but only half of it in the non-reference case.

Could you kindly clarify whether this scaling behavior is intentional or if there might be an error in our utilization of the software? 

Furthermore, we are keen to discuss additional feedback, queries, and requests we have regarding this version, and we would greatly appreciate to schedule a short call.
![1_ field on non-ref phantom](https://github.com/ncidosimetry/ncidoseforum/assets/22055904/8cb673f8-e5fd-4eca-9e4d-3fa007a49124)
![2_ 1_ field on ref phantom](https://github.com/ncidosimetry/ncidoseforum/assets/22055904/faf60e64-5852-40ce-8dff-6e320cbbac8c)
![3_ big field on ref phantom](https://github.com/ncidosimetry/ncidoseforum/assets/22055904/476d96c5-1e09-4a95-b66f-23c57a3ffe48)
![4_ big field on non-ref phantom](https://github.com/ncidosimetry/ncidoseforum/assets/22055904/36356ce0-218e-4815-b5c2-f4ffa1f34322)

Thank you for your attention and we look forward to your insights,

**Reply by choonsiklee** [reply]
Thank you for reporting this critical issue. We are fixing the bug and plan to release the next beta version shortly.

**Comment by choonsiklee** [comment]
New beta version 3.0.20240710 released! Please send us feedback.

**Comment by choonsiklee** [comment]
NCIRF Beta 3.0.20240718 released with some bugs fixed.

- 3.0.20240718
  - Bug fix: wrong Monte Carlo-based recalculated DAP set used for normalization
  - Bug fix: x-ray spectrum selection shifted
